test/compile/selection/multi.test.ts

```diff
@@ -155,9 +142,41 @@ describe('Multi Selection', () => {
     expect(assembleUnitSelectionData(model, data)).toEqual([
       {name: 'one_store'},
       {name: 'two_store'},
-      {name: 'thr_ee_store'},
-      {name: 'four_store'},
-      {name: 'five_store'}
+      {
+        name: 'thr_ee_store',
+        values: [
+          {
+            unit: '""',
+            fields: [{type: 'E', field: 'Horsepower'}],
+            values: [50]
+          }
+        ]
+      },
+      {
+        name: 'four_store',
+        values: [
+          {
+            unit: '""',
+            fields: [{field: 'Horsepower', channel: 'x', type: 'E'}, {field: 'Origin', channel: 'color', type: 'E'}],
+            values: [50, 'Japan']
+          }
+        ]
+      },
+      {
+        name: 'five_store',
+        values: [
+          {
+            unit: '""',
+            fields: [{type: 'E', field: 'Year'}, {type: 'E', field: 'Origin'}],
+            values: ['1970-02-02T05:00:00.000Z', 'Japan']
```

Comment:
Turns out Travis has a different time offset causing the checks to fail. I'll look into setting a particular timezone in jest for consistent testing across all devices.